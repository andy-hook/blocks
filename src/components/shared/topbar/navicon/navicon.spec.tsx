import React from "react"
import { renderWithTheme } from "@test-utils"

import Navicon from "./navicon"

describe("<Navicon />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme("dark", <Navicon onClick={jest.fn} />)
    expect(tree).toMatchSnapshot()
  })
})
