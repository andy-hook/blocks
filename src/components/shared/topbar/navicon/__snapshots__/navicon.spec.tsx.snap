// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navicon /> renders correctly 1`] = `
<button
  className="sc-bdVaJa IrEnj"
  onClick={[Function]}
>
  <svg
    className="sc-bwzfXH fKuZHR"
    viewBox="0 0 25 25"
  >
    <rect
      height="2"
      width="23"
      x="1"
      y="8"
    />
    <rect
      height="2"
      width="23"
      x="1"
      y="15"
    />
  </svg>
</button>
`;
